t/console-moodthemes.t

```diff
@@ -36,7 +36,7 @@ my $run = sub {
 
 # FIXME: make this neater.
 ok( $run->("moodtheme_list") =~ "Kanji Moods", "Got public theme" );
-ok( $run->("moodtheme_list 1") =~ "18x18 /img/mood/kanji/crazy.gif", "Got a theme" );
```

Comment:
With the re-dumping of moods.dat, the Kanji Moods is #11 now in a fresh db. Won't matter for old dbs.